graph/src/data/subgraph/mod.rs

```diff
@@ -1020,6 +1028,11 @@ impl UnvalidatedSubgraphManifest {
             });
 
         if let Some(graft) = &self.0.graft {
+            if *DISABLE_GRAFTS {
+                errors.push(SubgraphManifestValidationError::GraftBaseInvalid(
+                    "Grafting of subgraphs has been temporarily disabled".to_owned(),
```

Comment:
Hmm... It's not temporary if the node operator has deliberately and permanently disabled them. Just a thought, but nothing that should hold the PR back.

Reply:
Good point. Changed the message to `Grafting of subgraphs is currently disabled`